spark/src/test/scala/org/apache/spark/sql/delta/DeltaProtocolVersionSuite.scala

```diff
@@ -3427,6 +3427,23 @@ trait DeltaProtocolVersionSuiteBase extends QueryTest
       truncateHistory = truncateHistory)
   }
 
+  for (truncateHistory <- BOOLEAN_DOMAIN)
+  test(s"Protocol version downgrade with Table Features - include legacy reader features: " +
+    s"truncateHistory: ${truncateHistory}") {
+    val expectedFeatures =
+      Seq(DomainMetadataTableFeature, ChangeDataFeedTableFeature, ColumnMappingTableFeature)
+
+    // Although the protocol contains legacy features, the versions are only downgraded to
+    // table feature versions.
```

Comment:
This comment is stale?

Reply:
This is true in this PR but it won't be true when https://github.com/delta-io/delta/pull/2848 merges. Also, this test will have to be adapted once I rebase the latter PR. Anyway, I cleaned it up proactively.